scripts/firebase-config.ts

```diff
@@ -0,0 +1,58 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { cert, initializeApp } from 'firebase-admin/app';
+import { getFirestore } from 'firebase-admin/firestore';
+
+const SERVICE_ACCOUNT_PROD = 'serviceAccountProd.json';
+const SERVICE_ACCOUNT_DEV = 'serviceAccount.json';
+const DATABASE_URL_PROD = 'https://cornell-courseplan.firebaseio.com';
+const DATABASE_URL_DEV = 'https://cornelldti-courseplan-dev.firebaseio.com';
+
+function getDatabase(serviceAccount: string, databaseURL: string, appName?: string) {
+  return () => {
+    const credential = cert(serviceAccount);
+    const app = initializeApp(
+      {
+        credential,
+        databaseURL,
+      },
+      appName
```

Comment:
so glad you remembered to include this… I initially didn't have a name when I wrote the copy users script and I ran into issues when we tried having two apps initialized at the same time :)